src/view/seed.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import MainContent from '../component/main-content';
+import { CopyText, H1Text, Text } from '../component/text';
+import { SplitBackground } from '../component/background';
+import { GlasButton } from '../component/button';
+import { List } from '../component/list';
+import { color, font } from '../component/style';
+
+const styles = StyleSheet.create({
+  splitTop: {
+    flex: 1,
+    justifyContent: 'center',
+  },
+  title: {
+    padding: 20,
+    fontSize: font.sizeXXL + 2 * 6,
+    textAlign: 'center',
+  },
+  copyTxt: {
+    alignSelf: 'center',
+    textAlign: 'center',
+    fontSize: font.sizeL + 2 * 2,
+    lineHeight: font.lineHeightL + 2 * 4,
```

Comment:
If you need to multiply the fonts by a constant factor, that's probably a good sign that we need a new font size. Considering how large the seed view renders on the default viewport size though, I'm pretty sure we can do with the font sizes we currently already have.